### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by m4tty-d** [Horea94/Fruit-Images-Dataset#3]
Hey,

I don't know what I'm doing wrong, but every time I run the training after the first step the loss is already 0, and the accuracy is 1. (time: 18.3030 step: 1 loss: 0.0000 accuracy: 1.0000) I'm running the training with only the Banana dataset.

**Comment by Horea94** [comment]
Hello,

The neural network in this repo needs at least 2 separate classes in order to work properly.
If you are using a single class (in this case Banana), the network will automatically classify everything as a Banana, hence the 0 loss, 100% accuracy.

You can either use another fruit class beside banana or add your own class of "non bananas" with images and see if the issue persists. Note that if you add your own class, the images should be jpeg of size 100x100.

**Comment by m4tty-d** [comment]
Thank you for the fast reply! I added another class and the problem eliminated.

**Comment by m4tty-d** [comment]
I have another question which is not closely related to the other. I'd like to convert the training data to a single .pb file (opencv accept only this format). I tried to use the freeze_graph.py but it doesn't quite seem to work. Have you tried doing the same in the past?

**Comment by Horea94** [comment]
I did try to freeze the graph at some point. But for the purpose of this project, I did not need to do this. I used the freeze_graph.py too, the one found here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/freeze_graph.py

If you want to freeze the graph, you need to add this line
tf.train.write_graph(sess.graph_def, checkpoint_dir, 'graph.pbtxt')
next to this line in the train_net.py file
saver.save(sess, checkpoint_dir + 'model.ckpt')

After you have your 'graph.pbtxt', you can run the command:
freeze_graph.py 
--input_graph=graph.pbtxt 
--input_checkpoint=model.ckpt 
--output_graph=frozen.pb 
--output_node_names=softmax

Hope this helps.

**Comment by m4tty-d** [comment]
Thank you so much, it worked! I've been struggling with it for days.

**Comment by PrakharPrashantSingh** [comment]
Hey 

> Thank you so much, it worked! I've been struggling with it for days.

can you provide me a .pb model of this?
